test/server/Database.test.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from "bun:test";
+import { Database } from "bun:sqlite";
+import { getDB } from "../../server/database.ts";
+
+const testingDB = getDB();
```

Comment:
Use `beforeAll` instead of a global `const`.

Reply:
Eventually, this should also make an in-memory database instead of a file (i.e. with ":memory:" as the argument).